src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -0,0 +1,521 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import json
+import uuid
+import time
+import subprocess
+from subprocess import Popen, PIPE
+from base64 import b64encode
+
+from knack.util import CLIError
+from knack.log import get_logger
+from azure.cli.core.commands.client_factory import get_subscription_id
+from azure.cli.core.util import sdk_no_wait
+from azext_connectedk8s._client_factory import _graph_client_factory
+from azext_connectedk8s._client_factory import cf_resource_groups
+from azext_connectedk8s._client_factory import _resource_client_factory
+from msrestazure.azure_exceptions import CloudError
+from kubernetes import client as kube_client, config, watch  # pylint: disable=import-error
+from kubernetes.client.rest import ApiException  # pylint: disable=import-error
+from Crypto.IO import PEM  # pylint: disable=import-error
+from Crypto.PublicKey import RSA  # pylint: disable=import-error
+from Crypto.Util import asn1  # pylint: disable=import-error
+
+from .vendored_sdks.models import ConnectedCluster, ConnectedClusterAADProfile, ConnectedClusterIdentity
+
+logger = get_logger(__name__)
+
+
+# pylint:disable=unused-argument
+# pylint: disable=too-many-locals
+# pylint: disable=too-many-branches
+# pylint: disable=too-many-statements
+def create_connectedk8s(cmd, client, resource_group_name, cluster_name, location=None,
+                        kube_config=None, kube_context=None, no_wait=False, tags=None):
+    print("Ensure that you have the latest helm version installed before proceeding to avoid unexpected errors.")
+    print("This operation might take a while...\n")
```

Comment:
Consider using `logger.warning` instead of `print` so the **stdout** would only have the response in json format or other configured format and could be used in downstream commands in a script.